src/frontends/onnx/frontend/src/op/eye_like.cpp

```diff
@@ -90,14 +90,16 @@ Output<ngraph::Node> square_identity_matrix(const Output<ngraph::Node>& matrix_s
     const auto one_followed_by_zeros = std::make_shared<default_opset::Concat>(OutputVector{one, zeros}, 0);
 
     // The identity matrix as a 1D representation.
-    const auto one_int = default_opset::Constant::create(ngraph::element::i64, Shape{1}, {1});
-    const auto size_minus_one = std::make_shared<default_opset::Subtract>(matrix_size, one_int);
-    const auto one_d_data = std::make_shared<default_opset::Tile>(one_followed_by_zeros, size_minus_one);
-    const auto one_d_data_concat = std::make_shared<default_opset::Concat>(OutputVector{one_d_data, one}, 0);
+    const auto one_d_data = std::make_shared<default_opset::Tile>(one_followed_by_zeros, matrix_size);
+    const auto zero_int = default_opset::Constant::create(element::i64, Shape{1}, {0});
+    const auto eye_1d = std::make_shared<default_opset::Pad>(one_d_data,
+                                                             zero_int,
+                                                             std::make_shared<default_opset::Negative>(matrix_size),
+                                                             ov::op::PadMode::CONSTANT);
```

Comment:
The goal of adding Eye decomposition transformation is backward compatibility for the plugins without Eye op support, and direct usage of `ov::opset9::Eye` instead of subgraph in ONNX FE. 

Can this PR be updated with `ov::opset9::Eye` usage in ONNX EyeLike import? 
Then import fix shouldn't be be needed, and the final scenario will be tested.

Reply:
Changed.